<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telescopeConfigurationDialogForm</class>
 <widget class="QWidget" name="telescopeConfigurationDialogForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>590</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="BarFrame" name="LocationBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="stelWindowTitle">
      <property name="text">
       <string>Telescope Configuration</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="closeStelWindow">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxType">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>360</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>Telescope controlled by:</string>
   </property>
   <widget class="QRadioButton" name="radioButtonTelescopeLocal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>A telescope connected to this computer via a serial port and controlled directly by Stellarium.</string>
    </property>
    <property name="text">
     <string>Stellarium, directly through a serial port</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTelescopeConnection">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>A telescope controlled by an external application, either at this computer or at a remote machine.</string>
    </property>
    <property name="text">
     <string>External software or a remote computer</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBox" name="toolBoxSettings">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>360</width>
     <height>230</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabSpacing">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageDeviceSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>152</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Device Settings</string>
    </attribute>
    <widget class="QLabel" name="labelDeviceModelDescription">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDeviceModel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>300</width>
       <height>24</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLabel" name="labelSerialPort">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>180</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDeviceModel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>300</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Device model:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSerialPort">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxCount">
      <number>20</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageConnectionSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>152</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Connection Settings</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditTCPPort">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTCPPort">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelHost">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>210</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHostName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>200</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Host name or IPv4 address of the machine that hosts the telescope server</string>
     </property>
     <property name="text">
      <string>localhost</string>
     </property>
     <property name="maxLength">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageInterfaceSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>152</height>
     </rect>
    </property>
    <attribute name="label">
     <string>User Interface Settings</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditCircleList">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>300</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Up to ten decimal values in degrees of arc, separated with commas</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelCircles">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>300</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Circle size(s):</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCircles">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Show circles with fixed angular size around the telescope reticle</string>
     </property>
     <property name="text">
      <string>Use field of view indicators</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>360</width>
     <height>170</height>
    </rect>
   </property>
   <property name="title">
    <string>Telescope Properties</string>
   </property>
   <widget class="QLabel" name="labelConnectionDelay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection delay:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxTelescopeDelay">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>The approximate time it takes for the signals from the telescope to reach Stellarium.
Increase this value if the reticle is skipping.</string>
    </property>
    <property name="suffix">
     <string comment="seconds">s</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTelescopeName">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>70</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelTelescopeName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>300</width>
      <height>24</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>110</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxConnectAtStartup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>300</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Automatically attempt to start the telescope or establish connection when Stellarium starts</string>
    </property>
    <property name="text">
     <string>Start/connect at startup</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameOkCancelButtons">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>360</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButtonSave">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>28</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDiscard">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
